pkg/operator/ceph/object/config.go

```diff
@@ -42,6 +42,7 @@ caps osd = "allow rwx"
 	certKeyFileName                = "rgw-key.pem"
 	rgwPortInternalPort      int32 = 8080
 	ServiceServingCertCAFile       = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+	HttpTimeOut                    = 15
```

Comment:
How about this? Then the other places wouldn't need to multiply by time.Second.
```suggestion
	HttpTimeOut                    = time.Second * 15
```